packages/cli/src/commands/generate/cell/cell.js

```diff
@@ -31,48 +35,74 @@ const uniqueOperationName = async (name, index = 1) => {
   return uniqueOperationName(name, index + 1)
 }
 
+const getIdType = (model) => {
+  return model.fields.find((field) => field.isId)?.type
+}
+
 export const files = async ({
   name,
   typescript: generateTypescript,
   ...options
 }) => {
+  let cellName = name
+  let idType,
+    model = null
+  let templateNameSuffix = ''
+
+  if (options.list) {
+    cellName = options.list
+    templateNameSuffix = 'List'
+  } else {
+    // needed for the singular cell GQL query find by id case
```

Comment:
Note pretty, but needed for the destroy tests because for some reason they could never find the query engine binary.